src/Listbox.js

```diff
@@ -191,19 +192,17 @@ export const ListboxOption = {
   },
   render(h) {
     const isActive = this.context.activeItem.value === this.value
-    const isSelected = this.context.props.value === this.value
+    const isSelected = this.context.props.multiple
+      ? this.context.props.value.includes(this.value)
+      : this.context.props.value === this.value
 
     return h(
       'li',
       {
         attrs: {
           id: this.id,
           role: 'option',
-          ...(isSelected
-            ? {
-                'aria-selected': true,
-              }
-            : {}),
+          'aria-selected': isSelected ? 'true' : (this.context.props.multiple ? 'false' : null),
```

Comment:
I don't like multi-ternaries either 😄 

Will refactor it to use if statements, that make it more clear, what we are doing here.

We actually need the additional case, because a single select listbox just sets `aria-selected="true"` on the selected option. A multi select listbox additionally sets `aria-selected="false"` on all options that are not selected.

See the [WAI-ARIA specs](https://www.w3.org/TR/wai-aria-practices/#listbox_roles_states_props) for more details.